app/controllers/admin/modules.js

```diff
@@ -2,7 +2,16 @@ import Controller from '@ember/controller';
 
 export default Controller.extend({
   actions: {
-    submit() {
+    submit(moduleType, value) {
+      if (moduleType === 'ticketInclude') {
```

Comment:
All this is not required - use pipe actions to trigger a save along with mutating original values in the code when a user changes a setting. It will do that same thing without the need for this.

Reply:
I have pushed the changes with pipe actions.